02-oop-inheritance/Module pattern/script.js

```diff
@@ -0,0 +1,47 @@
+
+var inheritPrototype = function (childClass, parentClass){
+
+    var copyOfParent = Object.create(parentClass.prototype);
+    copyOfParent.constructor = childClass;
+    childClass.prototype = copyOfParent;
+};
+
+var Movie = (function () {
```

Comment:
i have already try something like this, but i have something wrong with the return statement, because i m getting "movie is not defined" error